lib/sendAnonymousCLIMetrics.js

```diff
@@ -0,0 +1,19 @@
+module.exports = send
+
+var assert = require('assert')
+var url = require('url')
+
+function send (uri, params, cb) {
+  assert(typeof uri === 'string', 'must pass registry URI')
+  assert(params && typeof params === 'object', 'must pass params')
+  assert(typeof cb === 'function', 'must pass callback')
+
+  var parsed = url.parse(uri)
+  assert(
+    parsed.protocol === 'http:' || parsed.protocol === 'https:',
+    'must have a URL that starts with http: or https:'
+  )
+
+  params.method = 'PUT'
+  this.request(uri, params, cb)
```

Comment:
Params here can't just be passed through to `request`.  What you're passing in the test isn't valid for `this.request`. Again, see `lib/dist-tags/set.js`, they would need to be JSON encoded and put into a `body` property.